### DISCUSSION TITLE
How to import a .osc file?

### DISCUSSION THREAD

**Discussion by rubyjiang18** [mtex-toolbox/mtex#959] [Ask Anything]
Greeting~
Sorry to ask this stupid question, but I have several .osc file from TSL OIM software. When I import it using mtex, i have this error message: 'Warning: more column data was passed in than expected. Check your column names make sense!' Does anyone know how to solve this issue
Thank!
Ruby

**Comment by lfgmorales** [comment]
Hi

What you probably have is an *.osc file that is incompatible with current version of MTEX, there are many versions of osc, the easiest is to export your ebsd map (on OIM) as a *.ang file, which is a text file that will open in mtex

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 14 Apr 2021, at 05:43, Ruby Jiang ***@***.***> wrote:

﻿

Greeting~
Sorry to ask this stupid question, but I have several .osc file from TSL OIM software. When I import it using mtex, i have this error message: 'Warning: more column data was passed in than expected. Check your column names make sense!' Does anyone know how to solve this issue
Thank!
Ruby

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/959>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN7DF54Y5PHM2H6ILZLTIUFPVANCNFSM424RDBMQ>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/959", "url": "https://github.com/mtex-toolbox/mtex/discussions/959", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by ralfHielscher** [comment]
You might send us the data and we will adapt the interface to read your file.

Ralf.
